0002/README.md

```diff
@@ -0,0 +1,167 @@
+---
+id: 0002
+title: Set the mood around releasing code
+status: ideation
+authors: Gianluca Arbezzano <gianarb92@gmail.com>
+---
+
+## Summary
+
+This proposal sets the expectation about what it means to release code part of
+the Tinkerbell organisation.
+
+## Goals and not Goals
+
+Goals are:
+
+1. What is a release
+2. Set a few rules that all the projects should follow
+3. Quality assertion.
+
+Not-Goals:
+
+1. How each project should be released
+2. Define who own releases
+3. Distribution via `apt`, `apk` and so on.
+
+## Content
+
+A release is a tested and stable point in time snapshot from a codebase.
+
+We follow the [Semantic Versioning (semver) 2.0.0](https://semver.org).
+Summarized as follows:
+
+Given a version number MAJOR.MINOR.PATCH, increment the:
+
+* MAJOR version when you make incompatible API changes,
+* MINOR version when you add functionality in a backwards compatible manner, and
+* PATCH version when you make backwards compatible bug fixes.
+
+Additional labels for pre-release and build metadata are available as extensions
+to the MAJOR.MINOR.PATCH format.
+
+### Schedule
+
+Every subproject should be able to follow its own cadence in terms of how often
+or regularly a new release should be cut, as long as it is documented and
+shared with the community.
+
+A new `MINOR` version for Tinkerbell should be released following a proper
```

Comment:
I think it is fair to use this two-three month release windows to bump the minor version due to a wealth of added features.  Until we reach 1.0.0, we may want to bump the minor version whenever we are forced to introduce breaking changes that would have otherwise been patch releases. Is that obvious or should it be pointed out?

Reply:
+1 I think this should likely call out the possibility of breaking changes landing in a minor release prior to v1.0

Reply:
Done

Reply:
The order of these paragraphs should be either "major", "minor", "patch", or possibly the reverse. I think it is confusing to lead with "minor".